docs/prover-submission-cost/prd.md

```diff
@@ -0,0 +1,134 @@
+
+In short, as a prover, I wish to:
+
+1. Collect blocks and transactions as the chain grows
+2. Run machines to prove transactions and roll them into an epoch proof
+3. Submit epoch proofs to the base-layer for a share of the rewards
+4. Profit???
+
+> [!WARNING]
+> **Multi-proof commentary**   
+> With the nature of the multi-proofs splitting rewards between all submitters and only paying out after. We won't know ahead of time if there is profit or not.
+
+# Requirements
+
+## Functional Requirements
+
+### Earmarked Fees
+
+- **What**: Fees earmarked for specific actions (e.g., proof submission) **MUST** be directed to the entity incurring the associated cost.
+- **Why**: To maintain fairness and economic incentive alignment between actors.
+- **Where**: Derived from the current imbalance (sequencer receives funds without bearing costs).
+
+### Protection from Excessive Multi-Proof Costs
+
+- **What**: Transaction fees charged to users **SHOULD** increase at most sub-linearly due to the introduction of multi-proof submissions compared to a single-proof sequencer setup.
+- **Why**: To prevent users from experiencing unexpectedly high fees resulting from internal design decisions (such as multiple proofs), ensuring network adoption and retention.
+- **Where**: Derived from concerns about potential abandonment during periods of low network utilization or initial adoption phases.
+
+## Non-functional Requirements
+
+### Prover profitability
+
+- **What**: Proving **SHOULD** be profitable in the presence of `<=M` submitters, even during periods of low activity.
+- **Why**: To ensure chain-growth and avoid avoid continuous pruning.
+- **Where**: Derived from other requirements on stable block production and looking at other chains and general economics.
+
+### Scalability
+
+- **What**: In the presence of `<=M` submitters their profit **SHOULD** scale with increasing transaction volume and overall network usage.
+- **Why**: To avoid creating incentives for provers and sequencers to collude, deliberately limiting network throughput to maximize their individual profits at the expense of network growth.
+- **Where**: Derived from concerns regarding potential economic collusion.
+
+### Inflation Bounds
+
+- **What**: The inflation of the staking asset **MUST** be less than 20% yearly
+- **Why**: Infinite inflation is not scalable.
+- **Where**: Common economic sends to limit inflation + @aminsammara for number.
+
+# Handling Tradeoffs
+
+When handling the tradeoffs, I believe we should step in the direction of "subsidise too much".
+
+For ignition there will be **no fees** since there are no transactions. If the provers cannot recoup their costs from the block rewards it is very limited who would be able **and** willing to participate.
+
+For alpha, it will also allow us to keep transaction fees low(er) while usage is low, making is easier for users to try the network.
```

Comment:
```suggestion
For alpha, it will also allow us to keep transaction fees low(er) while usage is low, making it easier for users to try the network.
```